apps/server/src/modules/room/api/service/room-board.service.ts

```diff
@@ -0,0 +1,91 @@
+import { BoardExternalReferenceType, ColumnBoard, ColumnBoardService } from '@modules/board';
+import { Injectable } from '@nestjs/common';
+import { EntityId } from '@shared/domain/types';
+import { RoomContentService } from '../../domain/service/room-content.service';
+
+@Injectable()
+export class RoomBoardService {
```

Comment:
same remark, not a blocker but why is room board and not more general room content and where to check if specified board id actually exists